code/__DEFINES/ai/ai_blackboard.dm

```diff
@@ -140,3 +140,39 @@
 #define BB_BASIC_MOB_REINFORCEMENT_TARGET "BB_basic_mob_reinforcement_target"
 /// The next time at which this mob can call for reinforcements
 #define BB_BASIC_MOB_REINFORCEMENTS_COOLDOWN "BB_basic_mob_reinforcements_cooldown"
+
+// bot keys
```

Comment:
this is gonna get ridiculous rather fast so i think it's time we make the `blackboard` folder in the `ai` folder, move the old file (the one you're currently editing) and then make a new file with the new keys in it.